tools/evtool/evtmain.cpp

```diff
@@ -39,6 +39,14 @@ static void usage(const char* tool, const CmdMap& commands, IBufferedSerialOutpu
 
 int main(int argc, const char* argv[])
 {
+    struct Cleanup
```

Comment:
Jake has a useful class in jutil.hpp he uses for this kind of pattern:

COnScopeExit cleanup([]() { cleanupConsole(); });

Reply:
Very useful, but not quite as you described. It either needs a complete lambda (not just the body), or since the signature conveniently matched expectations, just the function name.